site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -100,6 +75,11 @@ const TerminalPage: FC<
     applicationsHost,
   } = terminalState.context
   const reloading = useReloading(isDisconnected)
+  const shouldDisplayStartupWarning = workspaceAgent
+    ? ["starting", "starting_timeout", "start_error"].includes(
```

Comment:
This `start_error` is a permanent state when error is encountered. The workspace will never be `ready`. Perhaps we should show a different kind of warning in that case?

Reply:
Ahhh I see. I totally agree.